src/agent/libclusterfuzz/src/lib.rs

```diff
@@ -0,0 +1,28 @@
+// Rust generated by Microsoft, code originally from:
+//  https://github.com/google/clusterfuzz/blob/master/src/
+//      python/lib/clusterfuzz/stacktraces/constants.py
+//
+// Original Copyright:
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use anyhow::{Context, Result};
+use regex::RegexSet;
+
+mod generated;
+
+pub fn get_stack_filter() -> Result<RegexSet> {
```

Comment:
How about we (1) add a test to ensure the static regex set compiles, (2) make this function infallible?